### ISSUE TITLE
Plugin not working for me

### ISSUE THREAD

**Issue by cmdalbem** [cedricdelpoux/gatsby-remark-images-grid#1]
Hi!
I couldn't make this plugin work here.
Has anyone managed to make this work in conjunction with `gatsby-remark-images`? I'm wondering it might be a compatibility problem.

I've followed the guide and put it _before_ any other image plugins:

```javascript
{
  resolve: `gatsby-transformer-remark`,
  options: {
    plugins: [
      {
        resolve: "gatsby-remark-images-grid",
        options: {
          gridGap: "2rem",
        },
      },
      {
        resolve: `gatsby-remark-images`,
        options: {
          maxWidth: 1036,
          linkImagesToOriginal: false
        },
      },
      'gatsby-remark-smartypants',
      `gatsby-remark-reading-time`,
      'gatsby-remark-component',
    ],
  },
},
```

Here's how it looks like in the markdown code:

```markdown
```grid|4|test
![](./day2-dev.jpg)
![](./day2-dev2.jpg)
![](./day2-meeting.jpg)
![](./day2-team.jpg)
```

Here's how the final result looks like when inspected:

![image](https://user-images.githubusercontent.com/467471/46899727-75540d80-ce6d-11e8-9747-c240420472ae.png)

--

Full project code can be find here: https://github.com/cmdalbem/portfolio

**Comment by cmdalbem** [comment]
Sooo, I found out that adding this simple additional CSS to my site makes the image appear again:

```css
.gatsby-resp-image-wrapper {
  width: 100%;
}
```

It does solve my problem, but such a tweak wasn't necessary. Especially considering `gatsby-remark-images` is such a popular Gatsby plugin :)

**Comment by cedricdelpoux** [comment]
Fixed in `v1.1.0`
